libraries/AP_HAL_ChibiOS/hwdef/BlitzF7/README.md

```diff
@@ -0,0 +1,87 @@
+# iFlight BLITZ Whoop F7 AIO Flight Controller
+
+https://shop.iflight-rc.com/BLITZ-Whoop-F7-AIO-Pro1927
+
+The Beast H7 AIO is a flight controller produced by [iFlight](https://shop.iflight-rc.com/).
+
+## Features
+
+ - MCU: BGA-STM32F745, 216MHz
+ - Gyro: DPS310
+ - 16Mb Onboard Flash
+ - BEC output: 5V 2.5A
+ - Barometer: DPS310 or None
+ - OSD: AT7456E
+ - 5 UARTS: (UART1, UART2, UART3, UART4, UART7)
+ - I2C for external compass. UART3 pins are used for I2C (BRD_ALT_CONFIG=1)
+ - 5 PWM outputs (4 motors and 1 LED)
+
+## Pinout
+
+![BLITZ Whoop F7 AIO Board](blitz_f7_pinou.jpg "BLITZ Whoop F7 AIO")
```

Comment:
really need something that either lists all pins or a diagram with all pad labels shown